chromium_src/third_party/blink/common/origin_trials/origin_trials.cc

```diff
@@ -54,10 +53,7 @@ bool IsTrialDisabledInBrave(blink::mojom::OriginTrialFeature feature) {
       {
           blink::mojom::OriginTrialFeature::kAdInterestGroupAPI,
           blink::mojom::OriginTrialFeature::kDigitalGoods,
-          blink::mojom::OriginTrialFeature::kFencedFrames,
-          blink::mojom::OriginTrialFeature::kFledge,
           blink::mojom::OriginTrialFeature::kParakeet,
-          blink::mojom::OriginTrialFeature::kPrivacySandboxAdsAPIs,
```

Comment:
is this already disabled somewhere else?

Reply:
This was specifically for an origin trial. We do disable Privacy Sandbox overall via https://github.com/brave/brave-core/blob/master/components/privacy_sandbox/brave_privacy_sandbox_settings.h/cc. We also disable prefs in https://github.com/brave/brave-core/blob/master/browser/brave_profile_prefs.cc#L354-L357. I guess we could also explicitly disable `PrivacySandboxAdsAPIs` blink feature via renderer/brave_content_renderer_client.cc, do you think we need to?